modular_skyrat/modules/sec_haul/code/guns/guns.dm

```diff
@@ -1,1162 +0,0 @@
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/zeta.dmi'
-	righthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/righthand.dmi'
-	lefthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/lefthand.dmi'
-	icon_state = "zeta"
-	inhand_icon_state = "zeta"
-	accepted_magazine_type = /obj/item/ammo_box/magazine/internal/cylinder/zeta
-	fire_sound = 'modular_skyrat/modules/sec_haul/sound/revolver_fire.ogg'
-	fire_delay = 3
-
-/obj/item/gun/ballistic/revolver/zeta/give_manufacturer_examine()
-	AddElement(/datum/element/manufacturer_examine, COMPANY_BOLT)
-
-/obj/item/ammo_box/magazine/internal/cylinder/zeta
-	name = "\improper Zeta-6 cylinder"
-	desc = "If you see this, you should call a Bluespace Technician. Unless you're that Bluespace Technician."
-	ammo_type = /obj/item/ammo_casing/c10mm
-	caliber = CALIBER_10MM
-	max_ammo = 6
-
-/obj/item/ammo_box/revolver/zeta
-	name = "\improper Zeta-6 speedloader"
-	desc = "A speedloader for the Spurchamber revolver, chambered for 10mm Auto ammo."
-	icon_state = "speedloader"
-	ammo_type = /obj/item/ammo_casing/c10mm
-	max_ammo = 6
-	multiple_sprites = AMMO_BOX_FULL_EMPTY
-	caliber = CALIBER_10MM
-	start_empty = TRUE
-
-/obj/item/ammo_box/revolver/zeta/full
-	start_empty = FALSE
-
-/*
-*	REVOLUTION
-*/
-
-/obj/item/gun/ballistic/revolver/revolution
-	name = "\improper Revolution-8 revolver"
-	desc = "The Zeta 6's distant cousin, sporting an eight-round competition grade cylinder chambered for 9x25mm."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/revolution.dmi'
-	righthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/righthand.dmi'
-	lefthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/lefthand.dmi'
-	icon_state = "revolution"
-	inhand_icon_state = "revolution"
-	accepted_magazine_type = /obj/item/ammo_box/magazine/internal/cylinder/revolution
-	fire_sound = 'modular_skyrat/modules/sec_haul/sound/revolver_fire.ogg'
-	fire_delay = 1.90
-	projectile_damage_multiplier = 0.5
-
-/obj/item/gun/ballistic/revolver/revolution/give_manufacturer_examine()
-	AddElement(/datum/element/manufacturer_examine, COMPANY_BOLT)
-
-/obj/item/ammo_box/magazine/internal/cylinder/revolution
-	name = "\improper Revolution-8 cylinder"
-	desc = "If you see this, you should call a Bluespace Technician. Unless you're that Bluespace Technician."
-	ammo_type = /obj/item/ammo_casing/c9mm
-	caliber = CALIBER_9MM
-	max_ammo = 8
-
-/obj/item/ammo_box/revolver/revolution
-	name = "\improper Revolution-8 speedloader"
-	desc = "A speedloader for the Revolution-8 revolver, chambered in 9x25mm."
-	icon_state = "speedloader"
-	ammo_type = /obj/item/ammo_casing/c9mm
-	max_ammo = 8
-	multiple_sprites = AMMO_BOX_FULL_EMPTY
-	caliber = CALIBER_9MM
-	start_empty = TRUE
-
-/obj/item/ammo_box/revolver/revolution/full
-	start_empty = FALSE
-
-/*
-*	S.M.A.R.T. RIFLE
-*/
-
-/obj/item/gun/ballistic/automatic/smartgun
```

Comment:
What I said above about the smart gun, but here instead.

Reply:
The SMART rifle may have been unique but its unique trait was that it instantly won fights and is almost never used by anyone because it also instantly friendly fires anyone that dares to use it because of the insane bounce characteristics. Its been replaced with not only an infinitely more forgiving to use weapon, but also one that's a lot less bullshit for any parties involved.